deploy/complete/kubernetes/ingress/mushop-issuer.yaml

```diff
@@ -33,4 +33,11 @@ spec:
     solvers:
     - http01:
         ingress:
-          class: nginx
\ No newline at end of file
+          class: nginx
+---
+apiVersion: cert-manager.io/v1alpha2
```

Comment:
Is this actually used ? 
The manifests are all outdated at this point - IU feel we should remove them and replace them with an instruction to do a `helm template`

Reply:
The mushop -issuer.yaml is up to date already. What part do you think is not?

Reply:
i was talking about the other manifests in the `/kubernetes` directory. 

Is this manifest used by helm or the terraform script ?